charts/rapid-response/README.md

```diff
@@ -1,82 +1,88 @@
-# Rapid Response
+# Chart: Rapid Response
 
-See the [Rapid Response documentation](https://docs.sysdig.com/en/docs/sysdig-secure/investigate/rapid-response/) for details on the Rapid Response feature.
-Rapid Response allows security teams to connect to a remote shell within your environment to start troubleshooting and investigating an event using the
-commands they are already accustomed to, with the flexibility they need to run the security tools at their disposal, directly from the event alert.
+## Overview
 
+Rapid Response allows security teams to connect to a remote shell within your environment to troubleshoot and investigate an event using the commands they are already accustomed to, with the flexibility they need to run the security tools at their disposal, directly from the event alert. For more information on this feature, see [Rapid Response ](https://docs.sysdig.com/en/docs/sysdig-secure/investigate/rapid-response/).
 
+Use the [sysdig-deploy](../sysdig-deploy/README.md) parent chart to deploy the Rapid Response. Do not deploy subcharts directly.
 
-## Installing the Chart
+For installation instructions, see [Install Agent Components on Kubernetes](https://docs.sysdig.com/en/docs/installation/sysdig-secure/install-agent-components/kubernetes/).
```

Comment:
Correct link: https://docs.sysdig.com/en/docs/installation/sysdig-secure/install-agent-components/kubernetes/install-rapid-response/

Reply:
Thank you